caikit/core/data_model/dataobject.py

```diff
@@ -150,27 +183,14 @@ def decorator(cls: Type) -> Type[DataBase]:
         # Declare the merged class that binds DataBase to the wrapped class with
         # this generated proto class
         if isinstance(proto_class, type):
-            wrapper_class = _make_data_model_class(proto_class, cls)
+            setattr(cls, "_proto_class", proto_class)
+            cls = _make_data_model_class(proto_class, cls)
         else:
             enums.import_enum(proto_class, cls)
             setattr(cls, "_proto_enum", proto_class)
-            wrapper_class = cls
-
-        # Attach the proto class to the protobufs module
-        parent_mod_name = getattr(cls, "__module__", "").rpartition(".")[0]
```

Comment:
This was all in service of keeping the expected module structure for libraries with a "protobufs" module full of precompiled protos. Since we're moving away from that, this needed to go!